app/models/boarding.rb

```diff
@@ -0,0 +1,6 @@
+class Boarding < ApplicationRecord
+  has_one :ticket, dependent: :nullify
```

Comment:
why are we using `dependent nullify`?

Reply:
Apu, we are using `dependant: nullify` so that the foreign key is set to be NULL if a ticket is removed from the database.